package.json

```diff
@@ -20,7 +20,7 @@
   "peerDependencies": {
     "react": ">=16.0 <18.0",
     "react-dom": ">=16.0 <18.0",
-    "replicache": ">=4.0.1 <10.0 || >7.0.0-beta <7.0.0 || >8.0.0-beta <8.0.0 || >9.0.0-beta <9.0.0"
+    "replicache": ">=10.0.0-alpha.1"
```

Comment:
I don't understand why this change is required. An effect is that we can't release this package now until 10 is stable because it requires Replicache 10.